src/translations/en.json

```diff
@@ -889,7 +889,7 @@
         },
         "lovelace": {
           "caption": "Lovelace Dashboards",
-          "description": "Manage your Lovelace Dashboards",
+          "description": "Add and remove secondary Lovelace dashboards",
```

Comment:
secondary ? As you can build as many lovelace dashboards you want to. I think secondary is not the correct word.

Reply:
@SNoof85 good point. I'll change it.